src/renderers/webgl/WebGLProgram.js

```diff
@@ -517,6 +517,7 @@ function WebGLProgram( renderer, cacheKey, parameters, bindingStates ) {
 			( parameters.logarithmicDepthBuffer && parameters.rendererExtensionFragDepth ) ? '#define USE_LOGDEPTHBUF_EXT' : '',
 
 			'uniform mat4 modelMatrix;',
+			'uniform mat4 modelMatrixPrev;',
```

Comment:
Maybe moving this uniform definition to the meshvelocity shader for now? Only this shader requires it.

Reply:
Also, does it need to be a built-in material?

Reply:
It doesn't have to be.  I think it will be used similarly to MeshDepthMaterial and MeshNormalMaterial so I integrated it in the same way as those.

Reply:
Not arguing, but `MeshDepthMaterial` is used in core. This one, in only certain post-processing effects. And it has limited support, as you mentioned in the main post. Seems like it should be `VelocityShader`, instead.

But whatever you want is OK with me.

Reply:
Making it a built-in material seems overkill indeed.

Any chance you could refactor this to be in `examples/jsm` instead? Maybe `examples/jsm/materials`...

Reply:
I've placed it in "examples/jsm/shaders".   It seems to fit nicely there!